kberba/worldview
# Added more images and text to the getting started doc

## Added more images and text to the Getting Started doc

- Finished adding the details each section of the Getting Started doc

## How To Test

- For enhancements and features: What is the expected functionality being added/modified? How can a reviewer verify this?

No added functionality -- just added documentation. The reviewer can read through the document for clarity and see if it is appropriate to be added to the project. 

## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

@nasa-gibs/worldview
